bindings_ffi/src/mls.rs

```diff
@@ -367,6 +408,7 @@ impl FfiGroup {
             self.inner_client.as_ref(),
             self.group_id.clone(),
             self.created_at_ns,
+            self.added_by_address(),
```

Comment:
Why do you need to add this to every new group instance?

Reply:
Got extra clarity on Zoom and posted an addition comment outside of the bindings.